tests/performance/insert_select_squashing_variant.xml

```diff
@@ -19,7 +19,7 @@ CREATE TABLE src_variant_squash_performance
 (
     v Variant(Tuple(v1 Array(UInt64)), Tuple(v2 Array(UInt64)), Tuple(v3 Array(UInt64)), Tuple(v4 Array(UInt64)), Tuple(v5 Array(UInt64)))
 )
-ENGINE = Memory;
+ENGINE = Memory() SETTINGS max_rows_to_keep=1;
```

Comment:
This changes the meaning of the test, since now it will insert only one row into the `variant_squash_performance`